src/ckan_to_bigquery.py

```diff
@@ -1,14 +1,25 @@
 import os
 
 from google.cloud import bigquery
+from api_tracker import tables_in_query
+from google.cloud import storage
+from google.api_core import exceptions
+from google.api_core import retry
+from google.api_core.retry import if_exception_type
+
+from ckan.common import config
+import logging
+
+log = logging.getLogger(__name__)
 
 class Client(object):
-    def __init__(self, project_id, dataset):
+    def __init__(self, project_id, dataset, read_only_creds):
```

Comment:
if we are passing read_only_creds explicitly can we pass non-read only creds explicitly too?

Also can we please have a docstring giving info on the meaning of these variables (and where i get them from).

Reply:
> if we are passing read_only_creds explicitly can we pass non-read only creds explicitly too? 
yes, we can pass non-read only creds explicitly too.

>Also can we please have a docstring giving info on the meaning of these variables (and where i get them from).
sure

new commit with relevant changes - https://github.com/datopian/ckanext-datastore-bigquery/pull/2/commits/434cb85a4dd7ab9c9382ed0025e8c6c42ca6865f